q2dataflow/languages/wdl/templaters/action.py

```diff
@@ -69,8 +69,7 @@ def get_input_declarations_w_defaults(self, delimiter="\n        "):
         input_strs = self._get_delimited_param_str(True, True, delimiter)
 
         if input_strs:
-            result = f"""
-    input {{
+            result = f"""input {{
```

Comment:
The indentation here was actually correct in the original, but I made it match `output` choosing to keep the indentation from `{self.get_outputs()}` as it felt like a easier way to read the primary template.